app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/ResourceState.kt

```diff
@@ -0,0 +1,8 @@
+package com.kks.nimblesurveyjetpackcompose.model
+
+sealed class ResourceState<out T> {
+    object Loading : ResourceState<Nothing>()
+    data class Success<out T>(val successData: T) : ResourceState<T>()
```

Comment:
`successData` seems redundant, how about just `data`? 😄 
```suggestion
    data class Success<out T>(val data: T) : ResourceState<T>()
```